tests/converter/test_biosamples_converter.py

```diff
@@ -0,0 +1,124 @@
+import json
+import unittest
+from os.path import dirname
+
+from biosamples_v4.models import Sample, Attribute
+from typing import List
+
+from converter.biosamples import BioSamplesConverter
+
+
+class BioSamplesConverterTests(unittest.TestCase):
+
+    def setUp(self) -> None:
+        with open(dirname(__file__) + '/../resources/biomaterials.json') as file:
+            self.biomaterials = json.load(file)
+
+        self.biosamples_converter = BioSamplesConverter()
+        self.domain = 'self.test_domain'
+
+    def test_given_ingest_biomaterial_when_release_date_defined_in_project_converts_correct_biosample(self):
+        biomaterial = self.__get_biomaterial_by_index(0)
+
+        release_date = '2020-08-01T14:26:37.998Z'
+        biosample = self.__create_a_sample(release_date)
+
+        converted_bio_sample = self.biosamples_converter.convert(biomaterial, self.domain, release_date)
+
+        self.assertEqual(SampleMatcher(biosample), converted_bio_sample)
```

Comment:
Have you considered using the lovely assertions library [assertpy](https://github.com/assertpy/assertpy#objects) to save a lot of boilerplate assertion code?

Reply:
I will certainly use this library in the future. (Might also interest @ke4)